packages/core/src/lib/waku_store/index.ts

```diff
@@ -370,3 +377,20 @@ async function* paginate<T extends DecodedMessage>(
 export function isDefined<T>(msg: T | undefined): msg is T {
   return !!msg;
 }
+
+export async function createCursor(
+  message: string,
+  messageTimestamp: bigint,
+  contentTopic: string,
+  pubsubTopic: string = DefaultPubSubTopic
+): Promise<proto.Index> {
+  const contentTopicBytes = utf8ToBytes(contentTopic);
+  const messageBytes = utf8ToBytes(message);
+  const digest = sha256(Buffer.concat([contentTopicBytes, messageBytes]));
```

Comment:
`Buffer` is not available in the browser polyfill. Let's avoid polyfills to ensure the library is lightweight and easy to use.

There is `concat` function available in `@waku/byte-utils`